server/test/src/main/antlr4/org/eclipse/usecase/UseCasePreprocessor.g4

```diff
@@ -25,7 +25,7 @@ multiTokenError
    ;
 
 multiToken
-   : (word | copybookStatement | variableStatement | functionUsage | paragraphStatement | sectionStatement | subroutineStatement
+   : (identifiers | copybookStatement | variableStatement | functionUsage | procedureStatement | subroutineStatement
```

Comment:
why was it not replaced `words`?

Reply:
Good point, It seems we never use it like this in our test suit.